lib/fs/spifs/spifs.c

```diff
@@ -152,9 +152,11 @@ static status_t cursor_init(
 
 static uint8_t *cursor_get(cursor_t *cursor)
 {
+#if LK_DEBUGLEVEL > 1
     spifs_t *spifs = cursor->spifs;
 
     uint8_t *page_end = spifs->page + spifs->page_size;
+#endif
```

Comment:
I have no preference, and looking at the LK source found no precedence.

I will move it - it is probably more consistent.